salesforce_functions/context.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+
+from .data_api import DataAPI
+
+
+@dataclass(frozen=True, slots=True)
+class User:
+    """Information about the invoking Salesforce user."""
+
+    id: str
+    """The user's ID."""
+    username: str
+    """The name of the user."""
+    on_behalf_of_user_id: str | None
+    """The ID of the user on whose behalf this user is operating."""
+
+
+@dataclass(frozen=True, slots=True)
+class Org:
+    """Information about the invoking Salesforce organization and user."""
+
+    id: str
+    """The Salesforce organization ID."""
+    base_url: str
+    """The base URL of the Salesforce organization."""
+    domain_url: str
+    """The domain URL of the Salesforce organization."""
+    # TODO: This version is the version from pyproject.toml, not the cloud event.
+    # Should it have a clearer name/be omitted? Should we surface the version from the cloudevent?
+    api_version: str
```

Comment:
> Should we surface the version from the cloudevent?

For Java, the version from the cloud event is not available to the function. It is always the explicitly configured one.

Reply:
I've removed the `api_version` from `Context.org` in 13ec0a02e7671736cf6a9ea326a5b780ea0fedcb.

Configuring the client with the version from `project.toml` is tracked by GUS-W-11790278.